js/script.js

```diff
@@ -0,0 +1,59 @@
+const hambIcon = document.getElementById('hamburgerIcon');
+const closeIcon = document.getElementById('closeIcon');
+const menuItems = document.getElementById('myMenuItems');
+
+// Operation when SHOWING items
+const showItems = () => {
+  hambIcon.style.display = 'none';
+  hambIcon.style.visibility = 'hidden';
+  closeIcon.style.display = 'block';
+  closeIcon.style.visibility = 'visible';
+  menuItems.style.display = 'flex';
+  menuItems.style.visibility = 'visible';
+};
+// Operation when HIDING items
+const hideItems = () => {
+  closeIcon.style.display = 'none';
+  closeIcon.style.visibility = 'hidden';
+  hambIcon.style.display = 'block';
+  hambIcon.style.visibility = 'visible';
+  menuItems.style.display = 'none';
+  menuItems.style.visibility = 'hidden';
+};
+// Handling hamburgerMenu open/close behavior
+/* eslint-disable no-unused-vars */
+const hamburgerMenu = () => {
+  if (hambIcon.style.display === 'block') {
+    showItems();
+  } else {
+    hideItems();
+  }
+};
+// Handling hamburgerMenu open/close behavior when clicking item
+/* exported modal */
+const adaptHamburgerMenu = () => {
+  const menuItems = document.getElementById('myMenuItems');
+  if (menuItems.style.display === 'flex') {
+    hideItems();
+  }
+};
+/* eslint-enable no-unused-vars */
+/* When changing screen-size after closing the hamburger menu
+(therefore, hiding the "menuItems" list)
+so the items remain hidden on desktop.
+This shouldn't really be a problem since users wouldn't be browsing from one screen size to another,
+in the same session,
+but just to make the code function properly in such unrealistic scenarios, this fix is added */
+
+const x = window.matchMedia('(min-width: 768px)');
+
+x.onchange = (e) => {
+  const menuItems = document.getElementById('myMenuItems');
+  if (e.matches) {
+    menuItems.style.display = 'flex';
+    menuItems.style.visibility = 'visible';
+  } else {
+    menuItems.style.display = 'none';
+    menuItems.style.visibility = 'hidden';
+  }
+};
```

Comment:
- [ ] If you see the design then the background is totally white kindly fix it  to make it similar with figma design
### Your design
![your desing](https://user-images.githubusercontent.com/81727988/179810740-e3b27e66-0ccc-4206-8ab7-af73556d16a3.png)
### Figma design
![fffff](https://user-images.githubusercontent.com/81727988/179810801-c82e1d25-9f02-47b1-a5b9-6703b8017b9e.png)

Reply:
- [ ] As requested by the previous revieweer kindly match the Figma design as this is part of the requirements

Your design
![image](https://user-images.githubusercontent.com/81932478/179836948-d3d2d624-b500-4534-9f64-579a22ec45f2.png)

Figma design
![image](https://user-images.githubusercontent.com/81932478/179837090-c92de7b1-36f2-4f30-a722-cc8f30ce0c1f.png)

- Kindly remove the welcome text on the mobile menu, and ensure that the white background covers the fulllenght of the mobile view when you click the menu button as this is also part of the requirements.
- Check out my portfolio for ideas as I did the same one as well [Portfolio](https://github.com/Munsa1/Mobile-Version)